muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/communicator/CommunicatorAutoReplyController.java

```diff
@@ -0,0 +1,126 @@
+package fi.otavanopisto.muikku.plugins.communicator;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.ejb.Singleton;
```

Comment:
Import likely isn't used anymore. Eclipse has automatic import optimizer with Ctrl-Shift-o that removes unused imports.

Often if class files have the yellow triangle symbol they have some warnings like these that can be cleaned up.